Core/Categories/Category.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace Core.Categories
+{
+    public abstract class Category
+    {
+        public Category(string name, double increment, double capacity)
```

Comment:
Codacy found an issue: [Change the visibility of this constructor to 'protected'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=6003477)